### PR TITLE
Updated brewfile so that nerd font is found and installed using brew

### PR DESCRIPTION
**Pull request by micahvdk**
Simply updated the line within the brewfile so that it would look for the correct name of the font. It looks like they renamed the font within github which then affected Brew when trying to install.

### CODE REVIEW

File: dependencies/Brewfile [hunk 1]
```diff
@@ -100,7 +100,7 @@ brew "weaveworks/tap/eksctl"
 cask "cyberduck"
 cask "docker"
 cask "firefox"
-cask "font-firacode-nerd-font-mono"
+cask "font-fira-mono-nerd-font"
```
Comment (greg-cook) [inline dependencies/Brewfile:103]: I think it should actually be `font-fira-code-nerd-font`

Comment (greg-cook) [reply dependencies/Brewfile:103]: Fira Mono != Fira Code

Comment (micahvdk) [reply dependencies/Brewfile:103]: Ohhhh, I did see them both. But I figured that the one with "mono" in it was the more closely related.

Comment (micahvdk) [reply dependencies/Brewfile]: @greg-cook 

I've edited it as suggested above.

**Pull request merged**